units/Legion/Labs/leglab.lua

```diff
@@ -31,7 +31,7 @@ return {
 		sightdistance = 290,
 		terraformspeed = 500,
 		workertime = 100,
-		yardmap = "coooocoooooococcococcccooccccocccccc",
+		yardmap = "yooooy oooooo coccoc occcco occcco yccccy",
```

Comment:
remove `y` and replace with `o`
IMO this goes against the standardization your trying to achieve
yes there is a ruling on the footprint occpancy question. i think in this case that ruling should be ignored